boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -35,7 +35,8 @@
 		zephyr,console = &lpuart0;
 		zephyr,shell-uart = &lpuart0;
 		zephyr,uart-pipe = &lpuart0;
-		zephyr,canbus = &flexcan0;
+		zephyr,canbus0 = &flexcan0;
+		zephyr,canbus1 = &flexcan1;
```

Comment:
You are introducing a new naming for chosen nodes here. This will break other subsystems and samples relying on the current naming.

Why are these new chosen nodes needed?

Reply:
In the current Zephyr code, 
I can see some Device Trees have defined CAN nodes with node id's- **can<x>** while others have used **flexcan<x>**
In short, I there is no generic way of creating node id's for a CAN node.
Now, I wanted to create a generic application which uses multiple CAN nodes, so I thought that the application writer (based on the number of CAN PHYs on his board) could create **n** number of CHOSEN nodes like **zephyr_canbus<x>**.
I know that this will break other subsystems, and I can think more over it, but I feel that the original naming wasn't written with multiple CAN instances in mind.
@henrikbrixandersen I welcome your thoughts on this, please free to correct my understanding.

Reply:
The "node id" you refer to is called the nodelabel, which matches (or at least should match) what the given IP block is called in the SoC datasheet.

For out-of-tree applications, you can add whatever chosen nodes and/or devicetree aliases you need in a devicetree overlay. For in-tree  boards, samples, and tests we use the Zephyr-specific chosen nodes as documented here: https://docs.zephyrproject.org/latest/build/dts/api/api.html#zephyr-specific-chosen-nodes The rationale behind only supporting one CAN bus chosen node in Zephyr is that no in-tree code have yet needed to be able to support more than one CAN controller.

If you would like to propose a change to the naming of the Zephyr-specific chosen nodes, please open a separate RFC for this.

Reply:
Just to make sure I understood correctly -
I can write any chosen node names (even which breaks the rules mentioned in the link you provided) in the overlay of my out of tree application ?

Reply:
@sumitbatra-nxp ping

Reply:
I would take back this chosen node change, since I feel it makes more sense to choose just one instance of a subsystem (out of many)